scripts/in_container/prod/entrypoint_prod.sh

```diff
@@ -129,7 +129,7 @@ if [[ ${AIRFLOW_COMMAND} == "bash" ]]; then
 elif [[ ${AIRFLOW_COMMAND} == "python" ]]; then
    shift
    exec "python" "${@}"
+elif [[ ${AIRFLOW_COMMAND} == "airflow" ]]; then
+   shift
+   exec "airflow" "${@}"
 fi
-
-# Run the command
-exec airflow "${@}"
```

Comment:
Could we replace this all with just `exec "${@}"`?

Reply:
Line 115 is now not going to check the right thing anymore.

Reply:
@ashb so would the idea be that we just run the command as given?

Reply:
I think it needs to support both modes - this will break the chart otherwise!

Reply:
Oh you updated the chart.